docs/xdr/usecases/playbook/Add_UserAgent_in_comment.md

```diff
@@ -0,0 +1,39 @@
+# Enrich Alert with User agent in comment
+
+This use case describes how to enrich an alert with the User agent in comment
```

Comment:
```suggestion
This use case describes how to enrich the comments of an alert with the User agent extracted from the associated events.
```

Reply:
A suggestion, it's up to you. However, if you keep your sentence, you should add the final dot.